wiki/Tournaments/GTS/CGTS_2019/en.md

```diff
@@ -0,0 +1,199 @@
+---
+tags:
+- CGTS 2019
+- CGTS2019
+---
+
+# Continental Global Taiko Showdown 2019
+
+![CGTS 2019 Logo](img/logo.png)
+
+The **Continental Global Taiko Showdown 2019** (***CGTS 2019***) is an advanced worldwide double-elimination osu!taiko tournament hosted by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) and ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133). It is the first installment of the Continental Global Taiko Showdown and part of the Global Taiko Showdown series.
+
+## Tournament schedule
+
+### Regional phase
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-09-14/2019-10-20 |
+| Live Drawings | *to be announced* |
+| Qualifiers Stage | 2019-11-02/2019-11-03 |
+| Round of 32 | 2019-11-09/2019-11-10 |
+| Round of 16 | 2019-11-16/2019-11-17 |
+| Quarterfinals | 2019-11-23/2019-11-24 |
+| Semifinals | 2019-11-30/2019-12-01 |
+| Finals | 2019-09-07/2019-09-08 |
+| Grand Finals | 2019-09-14 |
+
+### Global phase
+
+| Event | Timestamp |
+| --: | :-- |
+| Qualifiers Stage | 2020-01-04/2020-01-05 |
+| Quarterfinals | 2020-01-11/2020-01-12 |
+| Semifinals | 2020-01-18/2020-01-19 |
+| Finals | 2020-01-25/2020-01-26 |
+| Grand Finals | 2020-02-01/2020-02-02 |
+
+## Prizes
+
+### Regional classement
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Exclusive 1st place banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Exclusive 2nd place banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Exclusive 3rd place banner |
+
+### Global classement
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, exclusive team banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Exclusive team banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Exclusive team banner |
+
+## Organisation
+
+The Continental Global Taiko Showdown 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
```

Comment:
not necessary or shorten to `The Continental Global Taiko Showdown 2019 is run by various community members.`